src/actions/guides/database/deleting_records.cr

```diff
@@ -0,0 +1,169 @@
+class Guides::Database::DeletingRecords < GuideAction
+  guide_route "/database/deleting-records"
+
+  def self.title
+    "Deleting records"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Deleting Records
+
+    ### Delete one
+
+    Deleting a single record is actually done on the [model](#{Guides::Database::Models.path}) directly.
+    Since each query returns an instance of the model, you can just call `delete` on that record.
+
+    ```crystal
+    user = UserQuery.find(4)
+
+    # DELETE FROM users WHERE users.id = 4
+    user.delete
+    ```
+
+    ### Bulk delete
+
+    If you need to bulk delete a group of records based on a where query, you can use `delete` at
+    the end of your query. This returns the number of records deleted.
+
+    ```crystal
+    # DELETE FROM users WHERE banned_at IS NOT NULL
+    UserQuery.new.banned_at.is_not_nil.delete
```

Comment:
You know what, this gives me an idea for adding `is_present`. The `isn_not_nil` always trips me up. I'm gonna open an issue for that https://github.com/luckyframework/avram/issues/344

But this is good to go as-is. Just a thought as I was reading through it